doc/doxygen/src/porting-boards.md

```diff
@@ -281,7 +281,7 @@ In this case some special considerations must be taken with the makefiles:
   `include $(RIOTBOARD)/foo-parent/Makefile.*include*`
 
 An example can be found in
-[`tests/external_board_native`](https://github.com/RIOT-OS/RIOT/tree/master/tests/external_board_native`)
+[`tests/external_board_native`](https://github.com/RIOT-OS/RIOT/tree/master/tests/external_board_native)
```

Comment:
There's something really wrong with this section in the documentation. Could please check?

Reply:
At least the additional ` in the URL is too much.

Reply:
> There's something really wrong with this section in the documentation. Could please check?

Hm, doxygen dosn't like doxygen commands inside markdown code blocks. I have changed these blocks to `@code ... @endcode`. There is no syntax highlighting but it works now.